README.md

```diff
@@ -582,8 +582,8 @@ class AdminController < ApplicationController
     super([:admin, scope])
   end
 
-  def authorize(record, query = nil)
-    super([:admin, record], query)
+  def authorize(record, query = nil, options = {})
```

Comment:
This isn't quite right: looking at the source this seems to be a specific keyword argument, rather than a generic list of options.